### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by wubinbai** [vivek081166/raw-audio-deep-learning#2]
Hi, with everything running okay, I was not able to get the good score you mentioned in the raw audio blog. What I have is high val loss and low val acc(definitely not around 0.9)

loss: 0.0629 - acc: 0.9939 - val_loss: 2.8613 - val_acc: 0.2941

Is this the same code you implemented in the blog? I am afraid something is not working good. Do you have any idea? thanks
